test2/task1/main.c

```diff
@@ -0,0 +1,162 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+char *maxDataFunction(char *data1, char *data2)
```

Comment:
```suggestion
char *getMaxDate(char *date1, char *date2)
```
Это и так функция, не нужно приписывать слова "function" к функции =)